docs/dev-guide/iframe.md

```diff
@@ -2,47 +2,101 @@
-    * **roomName**: (optional) name of the room to join.
-    * **width**: (optional) width for the iframe which will be created. If a number is specified it's treated as pixel units. If a string is specified the format is number followed by 'px', 'em', 'pt' or '%'.
-    * **height**: (optional) height for the iframe which will be created. If a number is specified it's treated as pixel units. If a string is specified the format is number followed by 'px', 'em', 'pt' or '%'.
-    * **parentNode**: (optional) HTML DOM Element where the iframe will be added as a child.
-    * **configOverwrite**: (optional) JS object with overrides for options defined in [config.js].
-    * **interfaceConfigOverwrite**: (optional) JS object with overrides for options defined in [interface_config.js].
-    * **noSSL**: (optional, defaults to false) Boolean indicating if the server should be contacted using HTTP or HTTPS.
-    * **jwt**: (optional) [JWT](https://jwt.io/) token.
-    * **onload**: (optional) handler for the iframe onload event.
-    * **invitees**: (optional) Array of objects containing information about new participants that will be invited in the call.
-    * **devices**: (optional) A map containing information about the initial devices that will be used in the call.
-    * **userInfo**: (optional) JS object containing information about the participant opening the meeting, such as `email`.
+```
 
-Example:
+## **Creating the Jitsi Meet API object**
+
+After you have installed the Meet API library, you must then create the Jitsi Meet API object.
+
+The Meet API object takes the following form:
+
+**`api = new JitsiMeetExternalAPI(domain, options)`**
+
+The API object constructor uses the following options:
+
+* **domain**: The domain used to build the conference URL (e.g., **`meet.jit.si`**).
+
+* **options**: The object with properties. 
+=
+  Optional arguments include:
+  
+    * **roomName**: The name of the room to join.
+    
+    * **width**: The created IFrame width.
+    
+    The width argument has the following characteristics:
+    
+    -- A numerical value indicates the width in pixel units.
+    
+    -- If a string is specified the format is a number followed by **`px`, `em`, `pt`, or `%`**.
+    
+    * **height**: The height for the created IFrame. 
+    
+    The height argument has the following characteristics: 
+    
+    -- A numerical value indicates the height in pixel units.
+    
+    -- If a string is specified the format is a number followed by **`px`, `em`, `pt`, or `%`**. 
+    
+    * **parentNode**: The HTML DOM Element where the IFrame is added as a child.
+    
+    * **configOverwrite**: The JS object with overrides for options defined in the [config.js] file.
+    
+    * **interfaceConfigOverwrite**: The JS object with overrides for options defined in the [interface_config.js] file.
+    
+    * **noSSL**: (Boolean) Indicates if the server should be contacted using HTTP or HTTPS. The default value is **`false`**.
```

Comment:
Drop this one, we have removed it.

Reply:
removed interfaceConfigOverwrite and noSSL

Reply:
interfaceConfigOverwrite is ok, it's just noSSL, sorry for not being specific!

Reply:
put interfaceConfigOverwrite back into documentation